volpiano_display_utilities/text_volpiano_alignment.py

```diff
@@ -92,12 +103,19 @@ def _postprocess_spacing(
         to associated text length
     """
     comb_text_and_vol_rev_spacing = []
-    for text_elem, vol_elem in comb_text_and_vol:
+    for text_elem, vol_elem in comb_text_and_vol[:-1]:
```

Comment:
what does `comb_text_and_vol[:-1]` represent? Could this be saved to a descriptive variable outside the for loop?